hello-word/src/components/Robots.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useEffect } from 'react';
+import UserCard from './UserCard';
+
+export default function Robots() {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    fetchUsers();
+  }, []);
+
+  const fetchUsers = () => {
+    fetch('https://random-data-api.com/api/users/random_user?size=10')
+      .then((response) => response.json())
+      .then((data) => setUsers(data));
+  };
+
+  return (
+    <div>
+    <button onClick={fetchUsers}>Fetch Random</button>
```

Comment:
Would you mind adding a little bit of styling to distinguish this button from normal text?
It can be way easier for the user to use it and even look cooler

Reply:
Also may you try the flipping the card on hover ?

Reply:
thanks for suggesting me those changes, I'll try to implement them according to my code workflow